cpp/src/plasma/client.cc

```diff
@@ -26,6 +26,7 @@
 #include <assert.h>
 #include <fcntl.h>
 #include <netinet/in.h>
+#include <stdbool.h>
```

Comment:
We can probably remove this.

Reply:
Merging this now, I'll remove the include in https://github.com/apache/arrow/pull/1939